preflight.sh

```diff
@@ -31,4 +31,4 @@ else
   fi
 fi
 echo "::debug ::Set PREFLIGHT_SHORT_LENGTH=$PREFLIGHT_SHORT_LENGTH"
-echo "::set-output name=PREFLIGHT_SHORT_LENGTH::$PREFLIGHT_SHORT_LENGTH"
+echo "PREFLIGHT_SHORT_LENGTH$PREFLIGHT_SHORT_LENGTH" >> $GITHUB_OUTPUT
```

Comment:
Missing the `=` symbol

```suggestion
echo "PREFLIGHT_SHORT_LENGTH=$PREFLIGHT_SHORT_LENGTH" >> "$GITHUB_OUTPUT"
```

Reply:
@rlespinasse Updated!